cipd_packages/codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -124,26 +130,30 @@ class FileCodesignVisitor {
   static final RegExp _notarytoolStatusCheckPattern = RegExp(r'[ ]*status: ([a-zA-z ]+)');
   static final RegExp _notarytoolRequestPattern = RegExp(r'id: ([a-z0-9-]+)');
 
-  static const String fixItInstructions = '''
+  static final String fixItInstructions = '''
```

Comment:
Because we're using the filenames from the enum now rather than hardcoding.

Reply:
interesting that dereferencing this is not a const expression, but oh well